MediaBrowser.XbmcMetadata/Parsers/SeriesNfoParser.cs

```diff
@@ -100,19 +100,8 @@ protected override void FetchDataFromXmlNode(XmlReader reader, MetadataResult<Se
                         break;
                     }
 
+                // Season names are processed by SeasonFromSeriesNfoParser
                 case "namedseason":
-                    {
```

Comment:
maybe add a `break` here so it doesn't re-check the key inside the base parser?
```suggestion
                case "namedseason":
                    break;
```